Makefile

```diff
@@ -37,10 +37,12 @@ LINT=golangci-lint
 MISSPELL=misspell -error
 MISSPELL_CORRECTION=misspell -w
 
-BUILD_INFO_IMPORT_PATH=github.com/signalfx/splunk-otel-collector/internal/version
+BUILD_INFO_IMPORT_PATH=go.opentelemetry.io/collector/internal/version
+BUILD_INFO_IMPORT_PATH_CORE=github.com/open-telemetry/opentelemetry-collector/internal/version
```

Comment:
Does it really make it work?

go.opentelemetry.io/collector is an alias for github.com/open-telemetry/opentelemetry-collector

I think it currently doesn't work because of `internal` part

Reply:
tested it, it works